models/stg_twilio__account_history.sql

```diff
@@ -1,5 +1,5 @@
---To disable this model, set the using_domain_names variable within your dbt_project.yml file to False.
-{{ config(enabled=var('using_twilio_account_history', True)) }}
+--To disable this model, set the using_domain_names variable within your dbt_project.yml file to False. 
+{{ config(enabled=var('using_twilio_account_history', True)) }} 
```

Comment:
Is there a reason we are allowing customers to disable this model? I imagine this is a pretty necessary model for the downstream transformations.

Reply:
Actually I noticed you added the disable config for **all** the models. Is there any reason in particular this was decided?

Additionally, I noticed the enabled configs were not added to the tmp models. If we do decide to include them, we should be sure to add the config to the tmp models. Otherwise the disabling will not function as expected.

Reply:
I added them in case not every customer uses that table. I suppose for account_history, that's pretty much a given. It's more an issue for call or message related table, if the customer doesn't use that certain service. I'll removed the disable config for account history and the others that will be massively used, will think through this a bit more

Reply:
added disable to the necessary tmp files

Reply:
It doesn't look like the disable configs were added to the tmp files.